scripts/machine-launch-script.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+set -e
```

Comment:
Would this be better as a `set -ex` so that you also get to what command runs and its args?